<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PPG_EDUKIT</class>
 <widget class="QMainWindow" name="PPG_EDUKIT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>581</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/vub.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>394</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="app1_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 20pt &quot;Miriam Mono CLM&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(216, 75, 33  , 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string>UART PPG PLOTTING</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="app2_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 20pt &quot;Miriam Mono CLM&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(40, 131, 211  , 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string> COMPARE PPG SOLUTIONS </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="app3_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 18pt &quot;Miriam Mono CLM&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(200, 200,200, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="app4_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 18pt &quot;Miriam Mono CLM&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(200, 200,200, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>534</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="show_pdf_list"/>
    <addaction name="separator"/>
    <addaction name="show_info_box"/>
   </widget>
   <addaction name="menu_Help"/>
  </widget>
  <action name="show_pdf_list">
   <property name="text">
    <string>&amp;Applications List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="show_info_box">
   <property name="text">
    <string>&amp;Info</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img/vub_image.qrc"/>
 </resources>
 <connections/>
</ui>
